catalog/opensearch/catalog-opensearch-endpoint/src/test/java/org/codice/ddf/opensearch/endpoint/query/OpenSearchQueryTest.java

```diff
@@ -668,6 +685,67 @@ public void testPolygonSpatialFilter() {
     }
   }
 
+  @Test
+  public void testGeometrySpatialFilter() {
+    OpenSearchQuery query = new OpenSearchQuery(0, 10, "relevance", "desc", 30000, FILTER_BUILDER);
+    query.addGeometrySpatialFilter(GEOMETRY_WKT);
+    Filter filter = query.getFilter();
+    assertThat(filter, notNullValue());
+    Intersects intersects = (Intersects) filter;
+    Literal literalWrapper = (Literal) intersects.getExpression2();
+    Object geometryExpression = literalWrapper.getValue();
+    assertThat(geometryExpression, instanceOf(GeometryImpl.class));
+    com.vividsolutions.jts.geom.Geometry polygon =
+        ((GeometryImpl) geometryExpression).getJTSGeometry();
+    assertThat(WKT_WRITER.write(polygon), is(GEOMETRY_WKT));
+  }
+
+  @Test
+  public void testMultipleSpatialFilter() {
+    OpenSearchQuery query = new OpenSearchQuery(0, 10, "relevance", "desc", 30000, FILTER_BUILDER);
+    query.addGeometrySpatialFilter(GEOMETRY_WKT);
+    query.addPolygonSpatialFilter(
+        "30.943,-120.032,35.039,-120.032,35.039,-110.856,30.943,-110.856,30.943,-120.032");
+    query.addBBoxSpatialFilter("-120.032,30.943,-110.856,35.039");
+    query.addPointRadiusSpatialFilter("117.3425", "33.9283", "5000");
+    query.addPolygonSpatialFilter(
+        "30.943,-120.032,35.039,-120.032,35.039,-110.856,30.943,-110.856,30.943,-120.032");
+
+    Filter filter = query.getFilter();
+    assertThat(filter, notNullValue());
+
+    // TODO: fix as OR filter after https://jira.di2e.net/browse/GSR-2130
```

Comment:
make ddf story for this

Reply:
updated https://codice.atlassian.net/browse/DDF-3857